engine/src/main/java/com/redhat/cloud/notifications/templates/TemplateService.java

```diff
@@ -25,6 +25,8 @@
 @ApplicationScoped
 public class TemplateService {
 
+    public static final String HCC_LOGO_TARGET = "Logo-Red_Hat-Hybrid_Cloud_Console-A-Reverse-RGB";
```

Comment:
This one is only used in the tests, could we move the variable there? (test code class if it's used by multiple tests)

Reply:
You are right, I moved it into TestHelper class which is much more appropriate.